ee/modules/030-cloud-provider-openstack/images/cloud-data-discoverer/discoverer.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2023 Flant JSC
+Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE
+*/
+
+package main
```

Comment:
What about discover_volume_types hook in openstack cloud provider module? Seems we don't want to have two places with service discovery.

Reply:
We decided move all service discoveries to standalone application. We will do it with in another task.